Test/OrochiUtils.h

```diff
@@ -14,8 +14,8 @@ class OrochiUtils
 		int x, y, z, w;
 	};
 
-	static oroFunction getFunctionFromFile( oroDevice device, const char* path, const char* funcName, std::vector<const char*>* opts );
-	static oroFunction getFunction( oroDevice device, const char* code, const char* path, const char* funcName, std::vector<const char*>* opts );
+	static oroFunction getFunctionFromFile( oroDevice device, const char* path, const char* funcName, const std::vector<const char*>& opts );
```

Comment:
Getting a pointer is to make it optional, setting 0 for the default parameter. Can we revert?

Reply:
I see. How about we use a `std::optional<std::vector<const char*>> optsIn  `  for this purpose and retain `if( optsIn )` ?
Or alternatively we can simply drop this PR since I just noticed that you have fixed the name in #8  to `SortKernel1` .. ?